src/Ledger/Ledger/Properties.agda

```diff
@@ -82,29 +87,535 @@ instance
 
 -- ** Proof that LEDGER preserves values.
 
-private variable
-  tx : Tx
-  Γ : LEnv
-  s s' : LState
-  l : List Tx
-
-FreshTx : Tx → LState → Set
-FreshTx tx ls = tx .body .txid ∉ mapˢ proj₁ (dom (ls .utxoSt .utxo))
-  where open Tx; open TxBody; open UTxOState; open LState
-
-LEDGER-pov : FreshTx tx s → Γ ⊢ s ⇀⦇ tx ,LEDGER⦈ s' → getCoin s ≡ getCoin s'
-LEDGER-pov h (LEDGER-V⋯ _ (UTXOW-inductive⋯ _ _ _ _ _ st) _ _) = pov h st
-LEDGER-pov h (LEDGER-I⋯ _ (UTXOW-inductive⋯ _ _ _ _ _ st))     = pov h st
-
-data FreshTxs : LEnv → LState → List Tx → Set where
-  []-Fresh : FreshTxs Γ s []
-  ∷-Fresh  : FreshTx tx s → Γ ⊢ s ⇀⦇ tx ,LEDGER⦈ s' → FreshTxs Γ s' l
-            → FreshTxs Γ s (tx ∷ l)
-
-LEDGERS-pov : FreshTxs Γ s l → Γ ⊢ s ⇀⦇ l ,LEDGERS⦈ s' → getCoin s ≡ getCoin s'
-LEDGERS-pov _ (BS-base Id-nop) = refl
-LEDGERS-pov {Γ} {_} {_ ∷ l} (∷-Fresh h h₁ h₂) (BS-ind x st) =
-  trans (LEDGER-pov h x) $
-    LEDGERS-pov (subst (λ s → FreshTxs Γ s l)
-                        (sym $ computational⇒rightUnique Computational-LEDGER x h₁)
-                        h₂) st
+module _ where
+
+  private variable
+    tx : Tx
+    Γ : LEnv
+    s s' : LState
+    l : List Tx
```

Comment:
Moved these inside a module since they clashed with variables in proof of ga deposits invariant below.